07.bitstream.semantics.md

```diff
@@ -1314,6 +1315,41 @@ coded video sequence.
 number of bits used to represent time_offset_value is equal to time_offset_length. When
 time_offset_value is not present, its value is inferred to be equal to 0.
 
+#### Metadata crop semantics
+
+When present the metadata_crop OBU applies starting at the next frame in the sequence 
+with matching temporal_id and spatial_id, and shall apply to all matching frames until the next Key Frame
+or metadata_crop OBU. The output picture should be cropped to the region as specified in the 
+cropping metadata OBU. When applied, the crop shall be after all normal decode operations as a 
```

Comment:
> When applied, the crop shall be after all normal decode operations as a post-processing step (after film grain synthesis in the output process).  This metadata has no effect on the decoding process.

Maybe it would be better to re-write like this:
> The crop shall be applied after all normal decode operations as a post-processing step.  This metadata information has no  effect on the decoding process.

Reply:
Used your new wording.